gef-ui/server/utils/helpers.js

```diff
@@ -297,25 +475,47 @@ const futureDateInRange = ({ day, month, year }, days) => {
      * 86400000 = 24 hours * 60 minutes * 60 seconds * 1000 ms
      * Number of ms in a day
      * */
-    range += (86400000 * days);
+    range += 86400000 * days;
     return input <= range;
   }
   return false;
 };
 
-const coverDatesConfirmed = (facilities) => (
-  facilities.items.filter(({ details }) => details.hasBeenIssued).length === facilities.items.filter(({ details }) => details.coverDateConfirmed).length
-);
+const coverDatesConfirmed = (facilities) =>
+  facilities.items.filter(({ details }) => details.hasBeenIssued).length === facilities.items.filter(({ details }) => details.coverDateConfirmed).length;
+
+/*
+   function returns true or false based on length of array
+   of facilities that have changed to issued from unissued
+*/
+const hasChangedToIssued = (application) => {
+  let changed = false;
+
+  const changedToIssued = facilitiesChangedToIssuedAsArray(application);
+
+  if (changedToIssued.length > 0) {
+    changed = true;
+  }
+
+  return changed;
+};
```

Comment:
Can we simplify above to this please?

const hasChangedToIssued = (application) => {
  const changedToIssued = facilitiesChangedToIssuedAsArray(application);
  return changedToIssued.length > 0;
};

Reply:
done